core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1985,6 +1985,21 @@ public void rename(AlluxioURI srcPath, AlluxioURI dstPath, RenameContext context
     }
   }
 
+  private boolean shouldPersistPath(String path) {
+    List<String> blacklist =
+        ServerConfiguration.isSet(PropertyKey.MASTER_PERSISTENCE_BLACKLIST)
+            ? ServerConfiguration.getList(PropertyKey.MASTER_PERSISTENCE_BLACKLIST, ",")
```

Comment:
shall we make `ServerConfiguration.getList` outside this local method to avoid this string operation every time?

Reply:
This breaks unit tests unfortunately. I could add another method to reset it if you think the performance is nontrivial.

Reply:
Updated